### PR TITLE
dev -> prod 10.12.0

### PR DESCRIPTION
**Pull request by jkonrath-postman**
TW-960 Doc Auto update/refresh of OAuth2 access tokens #4600
TW-906 API Governance and Security rule editor updates #4641
TW-458 Doc API definition to collection sync #4616
TW-714 Doc user shared auth, OAuth for on-prem, and multiple APIs to one repo #4640
TW-828 Doc OpenAPI 2.0 support for multi-file APIs and schema docs #4504
TW-953 Doc Improved UX for debugging test failures #4617
TW-933 Doc collection runs on resource usage dashboard #4645
Governance: TW-905 Improving the import library experience, TW-910 Granular control of governance rules #4635
TW-926 Simple import modal #4648
TW-986 HTTP clear response #4632
TW-900 Partner viewer role #4624
TW-962 Environment variable sorting #4625
TW-716 Domain capture improvements #4619
TW-867 Collection link and guest role #4603
updated homepage headers to comply with Sentence case rules #4639
TW-992 Remove bold from links #4643
MARKENG-2282 added trailing slash #4646
MARKENG-2196 Remove unused s3-deploy package #4649

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/sharing.md [hunk 1]
```diff
@@ -109,20 +111,59 @@ Anyone with this link has read-only access to the collection's JSON. The data th
...
+
+You can change an external user's role at the team and collection levels. You can also revoke access to viewing collections for new and existing external users.
+
+To learn how to change an external user's team role, see [Managing guests](/docs/administration/managing-your-team/managing-your-team/#managing-guests). To learn how an external user can request a role change, see [Requesting Editor role access for a collection as an external user](/docs/collaborating-in-postman/requesting-access-to-collections/#requesting-editor-role-access-for-a-collection-as-an-external-user).
+
+You can turn off the toggle next to **Allow people outside your team to view this collection**. When you turn this off, new external users can no longer be assigned the Guest role. External users already assigned the Guest role will continue to have access to the collection. If this is turned off for all collections an external user could access, they will be removed from the team and unable to access collections in the team. To remove external users already assigned the Guest role, see [Removing team members](/docs/administration/managing-your-team/managing-your-team/#removing-team-members).
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collaborating-in-postman/sharing.md:157]:
```suggestion
You can turn off the toggle next to **Allow Guests with the link to join your team and view collection**. When you turn this off, new external users can no longer be assigned the Guest role. External users already assigned the Guest role will continue to have access to the collection. If this is turned off for all collections an external user could access, they will be removed from the team and unable to access collections in the team. To remove external users already assigned the Guest role, see [Removing team members](/docs/administration/managing-your-team/managing-your-team/#removing-team-members).
```

File: src/pages/docs/collaborating-in-postman/sharing.md [hunk 2]
```diff
@@ -109,20 +111,59 @@ Anyone with this link has read-only access to the collection's JSON. The data th
...
+
+You can change an external user's role at the team and collection levels. You can also revoke access to viewing collections for new and existing external users.
+
+To learn how to change an external user's team role, see [Managing guests](/docs/administration/managing-your-team/managing-your-team/#managing-guests). To learn how an external user can request a role change, see [Requesting Editor role access for a collection as an external user](/docs/collaborating-in-postman/requesting-access-to-collections/#requesting-editor-role-access-for-a-collection-as-an-external-user).
+
+You can turn off the toggle next to **Allow Guests with the link to join your team and view collection**. When you turn this off, new external users can no longer be assigned the Guest role. External users already assigned the Guest role will continue to have access to the collection. If this is turned off for all collections an external user could access, they will be removed from the team and unable to access collections in the team. To remove external users already assigned the Guest role, see [Removing team members](/docs/administration/managing-your-team/managing-your-team/#removing-team-members).
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collaborating-in-postman/sharing.md:157]:
```suggestion
You can turn off the toggle next to **Allow Guests with the link to join your team and view collection**. When you turn this off, new external users can no longer be assigned the Guest role. External users already assigned the Guest role will continue to have access to the collection. If this is turned off for all collections an external user could access, they will be removed from the team and unable to access collections in the team.
```

File: src/pages/docs/designing-and-developing-your-api/importing-an-api.md
Comment (petewitcher-postman) [inline src/pages/docs/designing-and-developing-your-api/importing-an-api.md:1]: Line 3 should be `updated: 2023-03-15`

File: src/pages/docs/getting-started/importing-and-exporting-data.md
Comment (petewitcher-postman) [inline src/pages/docs/getting-started/importing-and-exporting-data.md:1]: Line 3 should be `updated: 2023-03-15`

File: src/pages/docs/sending-requests/managing-environments.md
Comment (petewitcher-postman) [inline src/pages/docs/sending-requests/managing-environments.md:1]: Line 4 should be `updated: 2023-03-15`

**Pull request merged**